### ISSUE TITLE
A query : when can be expect the codes to be posted here? 

### ISSUE THREAD

**Issue by sobalgi** [jhjacobsen/fully-invertible-revnet#1]
when can we expect the codes to be posted here? 

Just wanted check the implementation of DCT and what 'directly readout the logits from the DCT lowpass coefficients'.

Regards,
@sobalgi.

**Comment by jhjacobsen** [comment]
Hi, thanks for you interest.
Within a week it should be up here.

**Comment by sobalgi** [comment]
Thank you for your reply.

**Comment by sobalgi** [comment]
Hello, can I get an ETA on the code here?

**Comment by Kh4L** [comment]
Hi @jhjacobsen 

Great work and thanks for the paper!
Any update on the code availability? I am also curious about the DCT implem

Thank you

**Comment by pclucas14** [comment]
+ 1, am really curious to see the code!

**Comment by jhjacobsen** [comment]
Sorry to everyone, it was a busy couple of weeks recently. I am doing my best to get the code out asap. Thanks for your patience!

**Comment by pclucas14** [comment]
Sorry to bother you again with this, do you have a date for which we can expect the code ? I just want to know if I should try reimplementing from scratch, or wait a couple days until it appears :)

-Thanks in advance

**Comment by zxydi1992** [comment]
I am also interested in trying the network proposed in the paper. When it will be available?
